controllers/volumereplicationgroup_controller.go

```diff
@@ -228,6 +229,8 @@ func filterPVC(mgr manager.Manager, pvc *corev1.PersistentVolumeClaim, log logr.
 // +kubebuilder:rbac:groups=ramendr.openshift.io,resources=volumereplicationgroups/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=ramendr.openshift.io,resources=volumereplicationgroups/finalizers,verbs=update
 // +kubebuilder:rbac:groups=replication.storage.openshift.io,resources=volumereplications,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=replication.storage.openshift.io,resources=volumereplicationclasses,verbs=get;list;watch;update;patch
```

Comment:
(heads up) The file ramen/config/dr_cluster/rbac/role.yaml also has to be updated when roles change, as it is a partial copy of ramen/config/rbac/role.yaml to suit VRG reconciler needs (or ramen-dr-cluster needs)